src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.cs

```diff
@@ -252,6 +257,24 @@ protected override void OnApplyTemplate()
 
 		_shadowHost = skiaCanvas;
 		_canvas?.Children.Insert(0, _shadowHost!);
+
+		BindToShadowHost(_shadowHost);
+	}
+
+	private void BindToShadowHost(SKXamlCanvas skiaCanvas)
```

Comment:
probably not necessary, just nitpicking
but if you create a private event that notify the creation of `_shadowHost` which you would fire off from `OnApplyTemplate()`
you can refactor this stray method into BindToPaintingProperties()

Reply:
as it currently stands, it doesnt auto dispose when the control is deleted
the dispose pattern is practically useless here, 
unless OnApplyTemplate can fire multiple time which im not sure if that happens often

Reply:
ah I just understood that setting xx.Disposable = null dispose the previous one. That was a tricky one!